src/Stratis.Bitcoin.IntegrationTests/ProofOfStakeSteps.cs

```diff
@@ -74,14 +74,10 @@ public void PremineNodeStartsStaking()
 
         public void PremineNodeWalletHasEarnedCoinsThroughStaking()
         {
-            var network = this.nodes[this.PremineNode].FullNode.Network;
-            var premine = network.Consensus.ProofOfWorkReward + network.Consensus.PremineReward;
-            var mineToMaturity = network.Consensus.ProofOfWorkReward * 110;
-            var balanceShouldBe = premine + mineToMaturity;
             TestHelper.WaitLoop(() =>
             {
                 long staked = this.nodes[this.PremineNode].FullNode.WalletManager().GetSpendableTransactionsInWallet(this.PremineWallet).Sum(s => s.Transaction.Amount);
-                return staked > balanceShouldBe;
+                return staked > 0;
```

Comment:
Woah dude you are essentially breaking the POS minting integration test that ensures that we can stake coins!? This isn't right at all.

Reply:
Ok, this was very much my bad; I misread what this method was actually doing. I have added `includeImmature` flags to the other GetSpendable* methods so that the test logic can be used essentially unchanged here.